.github/workflows/build-templates.yml

```diff
@@ -207,6 +212,11 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
 
+      - name: Run nitro codegen
+        if: matrix.type == 'nitro-module'
+        run: |
+          yarn nitrogen
```

Comment:
We need to run this manually?

Also contributing guide needs to include the workflow in case there are any new steps - like running this (and when is it needed)